workstation/securedrop-client-0.8.1

```diff
@@ -0,0 +1,1631 @@
+user@disp4893:~$ git clone https://github.com/freedomofpress/securedrop-debian-packaging.git
```

Comment:
My dispVM wasn't called `sd-dev-dvm`, because I created a clean one on the fly. I can amend that.